src/oss-detect-backdoor/oss-detect-backdoor.csproj

```diff
@@ -20,7 +20,7 @@
     <None Remove="Resources\BackdoorRules\backdoor_signatures.json" />
     <None Remove="Resources\BackdoorRules\code_execution.json" />
     <None Remove="Resources\BackdoorRules\crypto_currency.json" />
-    <None Remove="Resources\BackdoorRules\data_exfilatration.json" />
+    <None Remove="Resources\BackdoorRules\data_exfiltration.json" />
```

Comment:
this is not important, but adding rules is frequent, we can publish it and download them if updated from oss-characterstics

Reply:
Something to consider if we end up updating the rules more frequently.